roofit/roofitcore/src/RooClassFactory.cxx

```diff
@@ -313,29 +303,25 @@ RooAbsPdf* RooClassFactory::makePdfInstance(const char* className, const char* n
   }
 
   // Create CINT line that instantiates specialized object
-  string line = Form("new %s(\"%s\",\"%s\"",className,name,name) ;
+  std::string line = Form("new %s(\"%s\",\"%s\"",className,name,name) ;
```

Comment:
Since this is a code modernisation, it is maybe better not using Form, although iy is ok in this case, but TString::Format or concatenate directly the string using std::string. 
See comment here on using Form: 
https://root-forum.cern.ch/t/question-on-tstring-form/27587/3

Reply:
Okay thanks for the comment! I updated it to use string concatenations